infrastructure/terraform/25-api-management-subscription.tf

```diff
@@ -19,6 +19,7 @@ resource "azurerm_api_management_subscription" "hmi_apim_subscription" {
 
 resource "azurerm_key_vault_secret" "subscription_key" {
   name         = "${var.prefix}-${var.product}-sub-key"
+  depends_on   = [azurerm_key_vault_access_policy.permissions, azurerm_api_management_subscription.hmi_apim_subscription]
```

Comment:
Please include a comment why an explicit dependency is required here: 

https://www.terraform.io/docs/configuration/resources.html#depends_on-explicit-resource-dependencies

Reply:
Good spot, thank you! It wasn't needed at all, removed.